### PR TITLE
autorandr: embed path to xrandr

### PR DESCRIPTION
**Pull request by ghost**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by dotlambda** [comment]
What's the benefit?

**Comment by gnidorah** [comment]
@dotlambda https://github.com/NixOS/nixpkgs/pull/26296#issuecomment-367386822

**Review by dotlambda — APPROVED** [review]
Btw, there's a `wrapPython` in the expression that isn't used anywhere.

**Comment by gnidorah** [comment]
@dotlambda Fixed, thanks!

**Comment by adisbladis** [comment]
@GrahamcOfBorg build autorandr

**Comment by adisbladis** [comment]
Thank you!

File: pkgs/tools/misc/autorandr/default.nix [hunk 1]
```diff
@@ -2,19 +2,23 @@
...
-  wrapPython = python3Packages.wrapPython;
   version = "1.4";
 in
   stdenv.mkDerivation {
     name = "autorandr-${version}";
 
     buildInputs = [ python ];
-    nativeBuildInputs = [ makeWrapper ];
+
+    # no wrapper, as autorandr --batch does os.environ.clear()
```
Comment (Mic92) [inline pkgs/tools/misc/autorandr/default.nix]: Not environ.clear, but it inherits PATH from other processes running in x11 sessions, as I discovered later.

**Pull request merged**